client/src/App.js

```diff
@@ -8,7 +7,7 @@ function App() {
       <GlobalStyle />
       <header className="App-header">
         <h1>SustainAtives</h1>
-        <img src={logo} className="App-logo" alt="logo" />
+        <a href="/.storybook">To Storybook</a>
```

Comment:
Link in deployment doesn't work :) 
```suggestion
        <a href="/storybook">To Storybook</a>
```

Reply:
Thanks! Already fixed now :)